api/algorithms/base/supervised.py

```diff
@@ -0,0 +1,22 @@
+from abc import abstractmethod
+from typing import Generic, TypeVar
+
+from algorithms.base.algorithm import Algorithm
+from schemas.configs.algorithms_configs import SupervisedAlgorithmsParams
```

Comment:
This should read:
from api.algorithms.base.algorithm import Algorithm
from api.schemas.configs.algorithms_configs import SupervisedAlgorithmsParams